dataplane/external/ext_dataplane.go

```diff
@@ -63,17 +76,26 @@ func StartExtDataplaneDriver(driverFilename string) (*extDataplaneConn, *exec.Cm
 	// Now the sub-process is running, close our copy of the file handles
 	// for the child's end of the pipes.
 	if err := toDriverR.Close(); err != nil {
-		cmd.Process.Kill()
+		_ = cmd.Process.Kill()
 		log.WithError(err).Fatal("Failed to close parent's copy of pipe")
 	}
 	if err := fromDriverW.Close(); err != nil {
-		cmd.Process.Kill()
+		_ = cmd.Process.Kill()
 		log.WithError(err).Fatal("Failed to close parent's copy of pipe")
 	}
 	dataplaneConnection := &extDataplaneConn{
 		toDataplane:   toDriverW,
 		fromDataplane: fromDriverR,
 	}
+
+	wg.Wait()
```

Comment:
This is wrong, this function didn't wait for the dataplane driver to finish before.  That's why we started background goroutines to copy the output to the log.

Reply:
Good point, fixed